src/Implementations/LinkedLists/cycle_detection.go

```diff
@@ -48,14 +44,12 @@ func HasCycle1(head *ListNode) bool {
 	return hasCycle
 }
 
-/*
-Solution 2: Using Floyd's algorithm
-Here you use 2 pointers, 1 moving at the speed of 1 and the other moving at the speed of 2.
-Finally the 2 pointers would meet each other if there is a cycle or the fast pointer would
-encounter null if it’s not a cycle.
-
-Time Complexity: O(n)	Space Complexity: O(1)
-*/
+// Solution 2: Using Floyd's algorithm
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported function HasCycle2 should be of the form "HasCycle2 ..."](https://app.codacy.com/manual/bumblebee211196/go-ds-daa/pullRequest?prid=5709817)